src/cclib/io/filewriter.py

```diff
@@ -44,7 +48,10 @@ def __init__(self, ccdata, jobfilename=None, terse=False,
         self.terse = terse
 
         self.pt = PeriodicTable()
-        self.elements = [self.pt.element[Z] for Z in self.ccdata.atomnos]
+        if hasattr(self.ccdata, 'atomnos'):
```

Comment:
This calculation has been moved to individual `filewriters`. `filewriter.__init__` should not depend upon any ccData attributes. PR: https://github.com/cclib/cclib/pull/389